src/dataflow/components/dataflow-program.tsx

```diff
@@ -97,6 +97,7 @@ interface IState {
   programDisplayState: ProgramDisplayStates;
   graphDataSet: DataSet;
   editorContainerWidth: number;
+  remainingTime: number;
```

Comment:
Consider renaming to `remainingTimeSeconds` or `remainingSeconds` just to clarify units.

Reply:
Good catch.  Should be clearer now.